### ISSUE TITLE
[Docs] Should I change the learning rate when using multi-node training?

### ISSUE THREAD

**Issue by mitming** [open-mmlab/mmdetection#9669]
### Branch

master branch  https://mmdetection.readthedocs.io/en/latest/

### 📚 The doc issue

It is not clear that whether we should change the learning rate when using multi-node training.

For example, let's make lr is 1e-4 for 8x2=16 batch size, when I using 8 nodes to train the model, each is 8x2=16 batch size, should I enlarge the learning rate to 8e-4 or just set 1e-4 on each machine?

**Comment by ZwwWayne** [comment]
It is determined by the overall batch size. If the batch size is not changed, then you do not need to change it.
